integrations/lever/fixtures/applyPosting.json

```diff
@@ -0,0 +1,16 @@
+{
+    "postId": "f8b7dd19-1d9f-40f1-8090-e4a62e989e0b",
+    "personalInformation": [
```

Comment:
Can this be improved. How is a user going to know what fields are required to apply to a posting. Can it be simplified to what is required to just be a key => value?
```
{
   "firstName": "John",
   "lastName": "Smith"
   "email": "foo@gmail.com"
}
```

Also can `eeoResponses` and `urls` be omitted from the user input? Then when going to the actual API we can fill in the empty object and empty array. Also `send_confirmation_email` should default to false unless specified. In general we should be doing more processing in the action to make it easier on the user instead of making the user have to go to the API docs to figure everything out.